hasher-matcher-actioner/hmalib/tests/test_threatexchange_from_sitepackages.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+import site
+import os
+import pathlib
+
+
+@unittest.skipIf(
+    "IGNORE_PY_TX_LOCAL" in os.environ,
```

Comment:
Hmm, unittest skips make me nervous, and gating behavior by environment variables seems like an especially messy way in the kong term.

Reply:
Ouch. This file should not exist. Have replaced with a shell script test.

Reply:
Will fix.